src/main/scala/com/microsoft/partnercatalyst/fortis/spark/transforms/topic/KeywordExtractor.scala

```diff
@@ -0,0 +1,38 @@
+package com.microsoft.partnercatalyst.fortis.spark.transforms.topic
+
+import com.microsoft.partnercatalyst.fortis.spark.transforms.Tag
+import org.apache.commons.collections4.trie.PatriciaTrie
+
+import scala.collection.mutable.ListBuffer
+
+@SerialVersionUID(100L)
+class KeywordExtractor(keyWords: Seq[String]) extends Serializable {
+  private val wordTokenizer = """\b""".r
+  private val keywordTrie = new PatriciaTrie[Unit]()
+  private val originalCase = Map[String, String](keyWords.map(s => (s.toLowerCase, s)): _*)
```

Comment:
Heh, actually I should just be storing the originally cased version as the Trie values if at all. @erikschlegel, do we want to preserve the casing of keywords when writing AnalyzedItems?